### PR TITLE
Using solana-trader-proto

### PR DESCRIPTION
**Pull request by tinystarinagalaxy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,29 @@ Refer to the `examples/` for more info.
 bloXroute Solana Trader API's interfaces are primarily powered by protobuf, so you will 
 need to install it for your system: https://grpc.io/docs/protoc-installation/
 
+In order to install local version of solana_trader_python_proto dependency
+
+you can build the solana-trader-proto/python directory using these commands
+
+
+```
+    cd /Users/mahmoud/Documents/workspace-go/solana-trader-proto/
```
Comment (atulsriv) [inline README.md]: remove the /Users/mahmoud

File: requirements.txt [hunk 2]
```diff
@@ -2,3 +2,4 @@
 build==0.8.0
 flake8==5.0.4
 pyre-check==0.9.15
+solana-trader-python-proto==0.0.6
```
Comment (atulsriv) [inline requirements.txt]: will we have to tag a new proto version during development to test changes?

**Review by atulsriv — DISMISSED** [review]

File: requirements.txt (hunk 2 above)
Comment (tinystarinagalaxy) [reply requirements.txt]: no, during developement, you dont need any version changes

File: README.md (hunk 1 above)
Comment (tinystarinagalaxy) [reply README.md]: done

File: .circleci/config.yml [hunk 3]
```diff
@@ -82,7 +88,27 @@ workflows:
...
+          cron: "0 0 12 * *"
+          filters:
+            branches:
+              only:
+                - develop
+    jobs:
+      - unit:
+          requires:
+            - init
+      - examples-mainnet:
```
Comment (aspin) [inline .circleci/config.yml]: run both testnet and mainnet here

File: README.md [hunk 4]
```diff
@@ -63,6 +63,32 @@ Refer to the `examples/` for more info.
...
+    
+    you can also push the package to pypi 
+   
+    python3 -m build
+    python3 -m twine upload --repository pypi dist/*
+    
+    and install the package locally now
+    python -m pip install ~/solana-trader-proto/python
+    
+
```
Comment (aspin) [inline README.md:88]: add an example for installing from `git` for CI builds

File: README.md [hunk 5]
```diff
@@ -63,6 +63,32 @@ Refer to the `examples/` for more info.
...
+you can build the solana-trader-proto/python directory using these commands
+
+
+```
+    cd ~/solana-trader-proto/
+    
+    upgrade the version in pyproject.toml
+    
+    rm -rf python/dist/
```
Comment (aspin) [inline README.md]: this step isn't necessary for making the protobuf files, only for packaging the protobuf, so move it down to line 81

**Review by aspin — CHANGES_REQUESTED** [review]
1 more request for installing from git

File: README.md (hunk 4 above)
Comment (tinystarinagalaxy) [reply README.md:88]: i have to figure this out later

File: README.md (hunk 5 above)
Comment (tinystarinagalaxy) [reply README.md]: moved

File: .circleci/config.yml (hunk 3 above)
Comment (tinystarinagalaxy) [reply .circleci/config.yml]: done

**Review by aspin — APPROVED** [review]

**Pull request merged**